extensions/wikia/PortableInfobox/services/Parser/MediaWikiParserService.php

```diff
@@ -31,12 +31,11 @@ public function parse( $wikitext ) {
 	 * class and method names.
 	 */
 	public function parseRecursive( $wikitext ) {
-		wfProfileIn(__METHOD__);
-		$parsed = $this->parse($wikitext);
-		$preprocessed = $this->parser->recursivePreprocess( $parsed, $this->frame );
-		$newlinesstripped = preg_replace( '|[\n\r]|Us', '', $preprocessed );
+		wfProfileIn( __METHOD__ );
+		$parsed = $this->parser->internalParse( $wikitext, false, $this->frame );
```

Comment:
As per offline discussion, we should fix the API response data (it is available to the community council and will be a public api once released, so needs to be adjusted)